packages/ragbits-document-search/tests/unit/test_local_file_source.py

```diff
@@ -0,0 +1,21 @@
+from pathlib import Path
+
+from ragbits.document_search.documents.sources import LocalFileSource
+
+TEST_FILE_PATH = Path(__file__)
+
+
+async def test_local_source_fetch():
+    source = LocalFileSource(path=TEST_FILE_PATH)
+
+    path = await source.fetch()
```

Comment:
```suggestion
    path = await source.fetch()
    
```
to follow the AAA convention